src/components/Frame/Frame.scss

```diff
@@ -86,15 +90,32 @@ $skip-vertical-offset: rem(10px);
     opacity: 1;
   }
 
+  &:hover {
+    background-color: var(--p-overlay);
+  }
+
   &:focus {
     border-radius: border-radius();
-    background-color: rgba(color('white'), 0.16);
+    background-color: var(--p-override-transparent, rgba(color('white'), 0.16));
     outline: none;
   }
 
+  &:active {
+    background-color: var(--p-overlay);
```

Comment:
Same with the hover style, let's keep the styles the same when global theming is **not** enabled.